packages/design-system-react-native/src/components/Box/README.md

```diff
@@ -168,11 +168,11 @@ The vertical padding (top and bottom) of the component. Use values from the BoxS
 
 ### `borderWidth`
 
-The border width of the component. Use values from the BoxSpacing scale (0-12).
+The border width of the component. Use only valid Tailwind CSS border width values (0, 1, 2, 4, 8).
```

Comment:
Tailwind border only has (0, 1, 2, 4, 8) unless we want to add custom values which I we can change later